File: fixtures/python_output/post_json_multiple_headers.py
import requests

headers = {
    'Content-Type': 'application/json',
    'X-API-Version': '200',
}

data = '{"userName":"username123","password":"password123", "authLoginDomain":"local"}'

response = requests.post('https://0.0.0.0/rest/login-sessions', headers=headers, data=data, verify=False)

File: test/fixtures/php-guzzle/post_json_multiple_headers.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->post('http://localhost:28139/rest/login-sessions', [
    'headers' => [
        'Content-Type'  => 'application/json',
        'X-API-Version' => '200'
    ],
    // 'body' => '{"userName":"username123","password":"password123", "authLoginDomain":"local"}',
    'json' => [
        'userName' => 'username123',
        'password' => 'password123',
        'authLoginDomain' => 'local'
    ],
    'verify' => false
]);
